src/cascadia/TerminalApp/App.cpp

```diff
@@ -52,11 +48,11 @@ static const std::array<std::wstring_view, 1> settingsLoadErrorsLabels {
 // Return Value:
 // - the localized string for the given type, if it exists.
 template<std::size_t N>
-static winrt::hstring _GetMessageText(uint32_t index, std::array<std::wstring_view, N> keys, ScopedResourceLoader loader)
+static winrt::hstring _GetMessageText(uint32_t index, std::array<std::wstring_view, N> keys, std::shared_ptr<ScopedResourceLoader> loader)

```

Comment:
this should just change to `ScopedResourceLoader&` -- there is no need to pass an entire shared pointer owned reference.

Reply:
changed